ratis-server/src/main/java/org/apache/ratis/server/raftlog/segmented/SegmentedRaftLogCache.java

```diff
@@ -145,6 +146,25 @@ public String toString() {
   }
 
   static class LogSegmentList {
+
+    static class LogSegmentListCacheInfo {
+      private final long totalCacheSize;
+      private final long cachedCount;
```

Comment:
Let's simply call it `size` and `count` and add javadoc.

Reply:
Updated according to the suggested patch diff.